src/offices/_single.html

```diff
@@ -91,7 +91,7 @@ <h1 class="h3">{{ top_story.head | safe }}</h1>
                     block__padded-top
                     block__border-top">
         <h1 class="h2">
-            {{ resource_title }}
+           {{ resource_title }}
```

Comment:
This isn't used anywhere. It's just a placeholder if we decide to set a title for the Resources block. Do you think we should just remove it or add a TODO?